### DISCUSSION TITLE
Welcome to Docker Minecraft Java Edition Server Discussions!

### DISCUSSION THREAD

**Discussion by itzg** [itzg/docker-minecraft-server#680] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, [see if there's an existing discussion to join](https://github.com/itzg/docker-minecraft-server/discussions) or [create a new discussion](https://github.com/itzg/docker-minecraft-server/discussions/new).

Also keep in mind that there is also a [Discord server](https://discord.gg/DXfKpjB) available for discussions.

**Comment by Echo128808** [comment]
Hi, do you know how I can send commands to the Minecraft server, like /weather clear?

**Reply by itzg** [reply]
Check out this section of the README

https://github.com/itzg/docker-minecraft-server#interacting-with-the-server

**Comment by MaverickTN** [comment]
I'm wanting to pass "-XX:ParallelGCThreads=6" to java for multi-threading.  I've tried -e JVM_XX_OPTS=ParallelGCThreads=6 and -e JVM_XX_OPTS=ParallelGCThreads:6 with no success. Any thoughts?

Nevermind:
The solution.
-e JVM_XX_OPTS="-XX:ParallelGCThreads=6"
